src/sfizz/Region.h

```diff
@@ -43,17 +43,7 @@ class RegionSet;
  *
  */
 struct Region {
-    Region(int regionNumber, const MidiState& midiState, absl::string_view defaultPath = "")
-    : id{regionNumber}, midiState(midiState), defaultPath(std::move(defaultPath))
-    {
-        ccSwitched.set();
-
-        gainToEffect.reserve(5); // sufficient room for main and fx1-4
-        gainToEffect.push_back(1.0); // contribute 100% into the main bus
-
-        // Default amplitude release
-        amplitudeEG.release = Default::ampegRelease;
-    }
+    Region(int regionNumber, const MidiState& midiState, absl::string_view defaultPath = "");
```

Comment:
If a control header sets `default_path` after this region is built, this region's `defaultPath` turns into a dangling pointer.

Reply:
This is wrong sure, I thought this was an std::string at some point (and it probably should be).

Reply:
There's 2 possible ways to fix this.
- should the region reference the path known to the synth, or
- copy the value at the time of instantiation?

ie. does a `default_path` opcode take effect on all regions, or just on the regions which appears after it?
```
<region> // A
<control> default_path=[..]
<region> // B
```

Reply:
I corrected it by copying it in the current state, to be on the safe side. To answer your question, it's only the regions that come after. If we wished to be efficient we could have a vector of "defaultPaths" in the current sfz file, with this string view referencing the proper one.